library/src/main/java/io/github/boguszpawlowski/composecalendar/header/DefaultWeekHeader.kt

```diff
@@ -0,0 +1,71 @@
+package io.github.boguszpawlowski.composecalendar.header
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowLeft
+import androidx.compose.material.icons.filled.KeyboardArrowRight
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.platform.testTag
+import io.github.boguszpawlowski.composecalendar.states.CurrentState
+import java.time.format.TextStyle.FULL
+import java.util.Locale
+
+@Composable
+public fun DefaultWeekHeader(
+  currentState: CurrentState,
+  modifier: Modifier = Modifier,
+) {
```

Comment:
Isn't this like a header for month / week depending on the situation? If this is the case, I think this name could be misleading.